aplib.net-demo/Assets/Scripts/Player/Inventory.cs

```diff
@@ -0,0 +1,115 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class Inventory : MonoBehaviour
+{
+    //TODO Add icon logic to Item class
+    //TODO Add hotbar Object which contains 4 subjobjects. They are linked to the inventory class and will display the right icons depending on inventory contents.
+    //TODO Add controls for using items to InputManager(?) class
+    //TODO Add controls for switching which item you're using to InputManager(?) class
+    //TODO Add logic for items with a duration and effect
+    Queue<Item> _itemList;
+    public float inventorySize;
+    public RawImage icon;
+    public Texture iconTexture;
+    public GameObject inventoryObject;
+    //here you would add all the possible items you can add, you add them to the inventory by calling PickUpItem method with the right item
```

Comment:
This comment is also a TODO comment I think? Same comment applies as before.

Reply:
No, it's a reminder of where you would put new items if you make them. if you want to make a potion, you make the prefab, add it there and then link them in the editor.